MIGRATION.md

```diff
@@ -386,6 +387,46 @@
 
 ## From version 7.x to 8.0.0
 
+### Tab addons are now routed to a query parameter
```

Comment:
This migration note should describe the URL structure before the change and after. The current content should be moved down to "Addon author changes" since it's really not end user facing.

Reply:
Also we can merge this PR as is and fix it in a follow-on PR, since time is tight